src/style.css

```diff
@@ -1,3 +1,86 @@
+.roboto-regular {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+  align-items: center;
+  font-weight: 500;
+}
+
+img {
+  display: block;
+  height: 40px;
+  width: 40px;
+  padding-left: 50px;
+}
+
+.nav {
+  justify-content: flex-end;
+  list-style: none;
+}
+
+.nav__link {
+  list-style: none;
+  color: black;
+  text-decoration: none;
+  display: block;
+  position: relative;
+  line-height: 60px;
+}
+
+.nav__item {
+  display: inline;
+  font-size: 12px;
+  font-weight: 500;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item :hover {
```

Comment:
There is a space between '.nav__item' and ':hover', which makes this selector target any child of '.nav__item' that is hovered, rather than '.nav__item' itself. Remove the space to ensure the hover effect applies directly to '.nav__item'.